packages/lib-content/src/screens/About/About.js

```diff
@@ -70,6 +70,8 @@ function AboutPage() {
         src='https://widget.freshworks.com/widgets/44000004375.js'
         async
         defer
+        crossOrigin='anonymous'
+        integrity='sha384-GpJCLE7VNj6jp2jskwQ5wnbyfRLB4jsiVzyCg1pJuMBnAqjylfzWNhQu3yKQeidZ' // Generated by https://www.srihash.org
```

Comment:
Unfortunately had to revert this change in https://github.com/zooniverse/front-end-monorepo/pull/6306 because zoo team members tested https://fe-root.preview.zooniverse.org/about#contact on multiple different browsers and devices, and could not open the Contact Us widget.

Reply:
Unfortunately, that does leave site users open to having their personal information hacked via `widget.freshworks.com`.

I had another look in Chrome, but the request is sending the `Origin` header, as expected, and `widget.freshworks.com` is responding with `Access-Control-Allow-Origin: *`, as expected. So I don't see anything that would block the script from loading with CORS enabled.

<img width=600 alt="Request headers, including Origin, for the Freshworks JS widget." src="https://github.com/user-attachments/assets/cdfc37b7-0b74-439b-98e3-00ae784dc0cc">

<img width=600 alt="Response headers for the Freshworks JS widget, including Access-Control-Allow-Origin" src="https://github.com/user-attachments/assets/06db0e49-b510-4f94-80cc-74adfa0c62c4">